packages/app/src/ui/pages/Settings/components/Backup/Backup.tsx

```diff
@@ -5,21 +5,48 @@ import Typography from "@mui/material/Typography";
 
 import { getEnabledFeatures } from "@src/config/features";
 
-export interface IAdvancedProps {
+export interface IBackupProps {
   isLoading: boolean;
   onDeleteIdentities: () => void;
   onGoToBackup: () => void;
+  onGoToUploadBackup: () => void;
 }
 
-const Advanced = ({ isLoading, onDeleteIdentities, onGoToBackup }: IAdvancedProps): JSX.Element => {
+const Backup = ({ isLoading, onDeleteIdentities, onGoToBackup, onGoToUploadBackup }: IBackupProps): JSX.Element => {
   const features = getEnabledFeatures();
 
   if (isLoading) {
     return <Box>Loading...</Box>;
   }
 
   return (
-    <Box data-testid="advanced-settings">
+    <Box data-testid="backup-settings">
+      {features.BACKUP && (
+        <Box sx={{ mb: 3 }}>
+          <Typography variant="h6">Backup</Typography>
+
+          <Typography color="text.secondary" variant="body2">
+            You can backup your storage containing identities, site approvals and password into a JSON file.
```

Comment:
We better should mention better `... into an encrypted backup JSON formatted file`.